Dot descriptions complete

ryanctan opened the pull request:
Fixes #138.

1. Description icons changed to question marks
2. Descriptions moved from sidebar to at top of active filter
3. Description text added

Also, label corrections that apparently were not pushed were made (explore.js)

ryanctan committed:
Dot descriptions complete

1. Description icons changed to question marks
2. Descriptions moved from sidebar to at top of active filter
3. Description text added

Also, label corrections that apparently were not pushed were made (explore.js)

epicfaace commented on public/explore/explore.pug:
```diff
@@ -54,9 +45,18 @@
+                          ng-mouseleave="(tooltip == dimension.field) && (tooltip = null)"
+                          ng-style="{'margin-left': '0px', 'margin-top': '-20px', display: 'block', opacity: tooltip == dimension.field ? '1': '0', 'z-index': tooltip == dimension.field ? 99: -1}"
+                    )
+                        //i(class="fa fa-times",style="top:10px;right:10px;position:absolute",ng-click="(tooltip == dimension.field) && (tooltip = null)")
```
Can you remove this line?

epicfaace reviewed (changes requested):
Looks good. Can you change it so that the tooltip comes to the _right_ of the question mark, rather than covering up the question mark? That way, it's a smoother experience for the user.

![image](https://user-images.githubusercontent.com/1689183/60850513-92233480-a1a4-11e9-92d6-812ee6c0c18c.png)

ryanctan committed:
tooltip polishing

1. Tooltip box shifted off of icon
2. Tooltip disappears when cursor leaves either icon or box

ryanctan commented:
Changes made!

ryanctan committed:
Delete app.js

ryanctan committed:
Revert "tooltip polishing"

This reverts commit e9e95d24c220872aeac97a3fb26fd18086f707a5.

ryanctan committed:
Merge branch 'dot-descriptions' of https://github.com/cestastanford/grandtour into dot-descriptions

ryanctan committed:
Update app.js

ryanctan committed:
Update app.js

ryanctan commented:
I'm gonna make a new branch, I accidentally committed non-relevant files.
